### PR TITLE
Add badges to 8.5 release notes

### PR DESCRIPTION
**Pull request by akeller**
## Description

Add badges, similar to #4198 enhancements!

Join the conversation [here](https://camunda.slack.com/archives/C026U8GBNSW/p1724784231042549).

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
Can I also have reviewers look at the badges for each epic in addition to a grammar/spelling review? This won't exactly match the component labels on the epic, but should cover the primary component AND include Self-Managed or SaaS if obviously relevant. 

Yes, there could be duplicates between the title and badge (Console Self-Managed, being the most obvious example), but I think this is ok because there is visual separation and styling between the title and the badges.

File: src/css/custom.css [hunk 1]
```diff
@@ -481,3 +481,9 @@ span.callout + p::after {
 .hero--primary {
   background-image: url("/static/img/hero-bg.png");
 }
+
+/* Remove badges from sidebar */
+a.table-of-contents__link.toc-highlight > span {
```
Comment (pepopowitz) [inline src/css/custom.css:486]:
```suggestion
a.table-of-contents__link.toc-highlight > span.badge {
```

non-blocking: any concerns about this accidentally hiding `span` elements that aren't badges? I don't know what those would be, but it also doesn't seem harmful to add a little more precision here.

**Review by pepopowitz — APPROVED** [review]
❤️ ❤️ ❤️ ❤️  I love this so much.

**Review by christinaausley — APPROVED** [review]
Really really like this!

**Comment by akeller** [comment]
Merging this to shift focus on #4198. Please open an issue or reach out to me via Slack if you have any additional feedback on 8.5 release notes or the overall badge project.

**Pull request merged**